benchmarks/go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/prometheus/common v0.7.0
 	github.com/prometheus/procfs v0.0.5
 	github.com/realshuting/multi-tenancy-benchmarks v0.0.0-20191028041724-a27250830445
+	github.com/realshuting/multi-tenancy/benchmarks v0.0.0-20200221171633-2b5c81967af0
```

Comment:
Shouldn't these point to this repo and not @realshuting's fork?

Reply:
I don't think any changes were supposed to be made to the go.mod and go.sum files. I will revert the,